app/helpers/user_info.rb

```diff
@@ -6,4 +6,18 @@ def current_user
   def self.current_user=(user)
     Thread.current[:user] = user
   end
+
+  def self.all_users
+    all_userpool_users = SelfService.service(:cognito_client).list_users(
+      user_pool_id: "eu-west-2_mQcoeYjGQ"
+    ).users
+    all_users = {}
+    all_userpool_users.each do |user|
+      attribs = user.attributes.map { |attribute|
+        [attribute.name, attribute.value]
+      }.to_h
+      all_users[attribs["sub"]] = attribs["given_name"] + " " + attribs["family_name"]
+    end
+    all_users
```

Comment:
```suggestion
    all_userpool_users.map { |user|
      attribs = get_user_attributes(user)
      [attribs["sub"], "#{attribs["given_name"]} #{attribs["family_name"}"]
    }.to_h
```

I wonder whether that would work . . . ??